packages/redux_persist_flutter/pubspec.yaml

```diff
@@ -1,18 +1,19 @@
 name: redux_persist_flutter
 description: Redux Persist Flutter Integration
-version: 0.6.0-rc.1
+version: 0.6.0-save-sync-2-SNAPSHOT
 homepage: https://github.com/Cretezy/redux_persist/tree/master/packages/redux_persist_flutter
 author: Charles Crete <charles@cretezy.com>
 
 environment:
-  sdk: ">=2.0.0-dev <2.0.0"
+  sdk: ">=2.0.0-dev <3.0.0"
```

Comment:
This is odd. I remember doing this change. I might have an unpushed commit on my desktop